File: 프로그래머스/0/181879. 길이에 따른 연산/길이에 따른 연산.java
class Solution {
    public int solution(int[] num_list) {
        
        // 리스트의 길이가 11 이상이면 리스트에 있는 모든 원소의 합
        // 10 이하이면 모든 원소의 곱
        int answer;
        if (num_list.length >=11){
            answer = 0;
            for(int num : num_list){
                answer += num;
            }
        
        }
        else {
            answer = 1;
            for(int num : num_list){
                answer *= num;
            }           
        }
        return answer;
    }
}

File: 프로그래머스/0/181879. 길이에 따른 연산/길이에 따른 연산.py
def solution(num_list):
    # 리스트의 길이가 11 이상이면 리스트에 있는 모든 원소의 합
    # 10 이하이면 모든 원소의 곱
    
    if len(num_list) >= 11:
        return sum(num_list)
    else:
        result=1
        for i in num_list :
            result *= i
        return result